Install_or_Update.bat

```diff
@@ -26,7 +26,7 @@ if not exist %MY_CONDA_ENV_FILE% (
 
 call "!MY_CONDA!" activate base ^
   && call "!MY_CONDA!" env create --force -n %ENV_NAME% --file %MY_CONDA_ENV_FILE% ^
-  && call "!MY_CONDA!" run -n %ENV_NAME% pip install -e .
+  && call "!MY_CONDA!" run -n %ENV_NAME% pip install -e .[dash]
```

Comment:
this line is usually to install the package itself from file. What does adding [dash] after the . do?

Reply:
I have declared qt and dash packages as optional in group "dash", so that `peak-finder` (or other packages using `geoapps-utils`) can depend on `geoapps-utils` without these indirect dependencies, and declare the dependencies as their own.
This is a trick, for conda-lock not to look for Qt and Dash on PyPI (and fail).
We will need a better way as we multiply repositories. I believe having them all in conda will help. Even if pyproject points to git branches during development, we would then be able to tell conda-lock not to look on PyPI (fetching only from git and conda).